### ISSUE TITLE
Notation 1.1.0 support

### ISSUE THREAD

**Issue by zosocanuck** [notaryproject/notation-action#52]
Getting this error when attempting to use the action while specifying the latest version 1.1.0:

```
Run notaryproject/notation-action/setup@v1
Error: Error: official Notation CLI release does not support version 1.1.0
```

**Comment by shizhMSFT** [comment]
Update is patched by #53. We need to publish a new release `v1.0.1` to ship the patch.

**Comment by shizhMSFT** [comment]
Close as `notation` v1.1.0 supported as added by the release [v1.0.1](https://github.com/notaryproject/notation-action/releases/tag/v1.0.1).

@zosocanuck Please create a new issue if there is any problem with the release.
